index.bs

```diff
@@ -788,6 +789,32 @@ the {{makeCredential()}} or {{getAssertion()}} method. It mirrors the fields of
 </div>
 
 
+### Credential Transport enumeration (enum <dfn enum>ExternalTransport</dfn>) ### {#transport}
+
+<pre class="idl">
+    enum ExternalTransport {
+        "USB",
+        "NFC",
+        "BLE"
+    };
```

Comment:
When we defined BT in the U2F JavaScript API, we believed that folks would build Authenticators using BR/EDR (or what we call "bluetooth classic").  However, that never came to be and we removed that from the Bluetooth specification.  Said another way "bt" is meaningless in U2F.  I'm trying to avoid making that mistake in webauthn :)

I'll make everything lowercase.